src/components/WeekDayPicker/index.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useUniqueIdentifier } from '../../libs/hooks';
+import RequiredAsterisk from '../RequiredAsterisk';
+import RenderIf from '../RenderIf';
+import WeekDayItems from './weekDayItems';
+import { StyledFieldset, StyledLabel, StyledTextError } from './styled';
+
+const WeekDayPicker = props => {
+    const {
+        id,
+        name,
+        value,
+        label,
+        hideLabel,
+        disabled,
+        required,
+        readOnly,
+        multiple,
+        error,
+        onChange,
+        className,
+        style,
+    } = props;
+
+    const defaultFieldsetName = useUniqueIdentifier('week-day-items');
+    const fieldsetName = name || defaultFieldsetName;
+
+    return (
+        <StyledFieldset className={className} style={style} id={id}>
+            <RenderIf isTrue={!hideLabel}>
+                <StyledLabel>
```

Comment:
You don't need to create a new Label element,  you should reuse the Label we have on the Input component instead of creating another new with the same styles